frontend/src/metabase/static-viz/components/LineAreaBarChart/LineAreaBarChart.tsx

```diff
@@ -29,21 +29,21 @@ const LineAreaBarChart = ({
       color: getColor("text-light"),
       ticks: {
         color: getColor("text-medium"),
-        fontSize: 11,
+        fontSize: 12,
       },
       labels: {
         color: getColor("text-medium"),
-        fontSize: 11,
+        fontSize: 14,
         fontWeight: 700,
       },
     },
     legend: {
-      fontSize: 13,
-      lineHeight: 16,
+      fontSize: 16,
+      lineHeight: 20,
```

Comment:
Since we're defining the lineheight in pixel units, I try to maintain the lineheight ratio (16/13 = 1.23) and 1.23*16 = 19.68, so I round it up to 20px.